src/main/java/org/veupathdb/service/eda/ds/plugin/correlation/AbstractCorrelationBipartiteNetwork.java

```diff
@@ -48,6 +48,7 @@ protected void writeResults(OutputStream out, Map<String, InputStream> dataStrea
 
     CorrelationStatsResponse stats = getComputeResultStats(CorrelationStatsResponse.class);
     Number correlationCoefThreshold = getPluginSpec().getCorrelationCoefThreshold() != null ? getPluginSpec().getCorrelationCoefThreshold() : 0.2;
+    Number pValueThreshold = getPluginSpec().getSignificanceThreshold() != null ? getPluginSpec().getSignificanceThreshold() : 0.05;
```

Comment:
I see that the response from this plugin does not contain any threshold values in `bipartitenetwork.config` - therefore the client doesn't know which default values were used (e.g. 0.2 and 0.05 above) if none were provided in the request.

Is this worth sorting out? I don't know. Just wanted to mention it!

Reply:
i like that suggestion. ill file a ticket for it i think. https://github.com/VEuPathDB/EdaDataService/issues/340